arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -174,19 +215,19 @@
             "type": "string"
           },
           {

```

Comment:
Please add descriptions
patch: "Updates a container registry with the specified parameters."
resourceGroup: "The name of the resource group within the user's subscription."
registryName: "The name of the container registry."
registryUpdateParameters: "The parameters to provide for the updated container registry."